app/helpers/templates_helper.rb

```diff
@@ -46,4 +46,19 @@ def locked_warning(template)
 
     alert(:class => 'alert-warning', :text => warning_text.html_safe)
   end
+
+  def self.template_help_text
+    {
+      "PXELinux" => N_("Deployed to the TFTP server to ensure the Host boots the correct installer using pxelinux.0 file"),
+      "PXEGrub" => N_("Deployed to the TFTP server to ensure the Host boots the correct installer using grub/bootx64.efi file"),
+      "PXEGrub2" => N_("Deployed to the TFTP server to ensure the Host boots the correct installer using grub2/grubx64.efi file"),
+      "iPXE" => N_("Used in {g,i}PXE environments in place of PXELinux"),
```

Comment:
gPXE is super old, I'd go with iPXE here.

Reply:
Sure, I'll make that change as well. Thanks!